distribution/ddf-common/src/main/resources-filtered/etc/system.properties

```diff
@@ -84,14 +84,21 @@ org.codice.ddf.system.registry-id=
 # Uncomment or comment the properties based on the desired configuration:
 #
 
-# Solr Server Provider
+# Solr Server Authentication
+solr.useBasicAuth=true
+solr.username=ddf
+solr.password=changeit
```

Comment:
❗️ We can't have a plain text password for the DB in a flat file. This is going to be a finding. I think what we could do instead is put the password into a metatype configuration. It will then be encrypted automatically when saved to disk and we can still retrieve the plain text value when needed. This also provides an easier mechanism to update the password automatically upon install, as is talked about below.

Reply:
I'll get to work on it.